ci-operator/step-registry/azure/provision/vnet/azure-provision-vnet-commands.sh

```diff
@@ -74,7 +97,11 @@ VNET_BASE_NAME="${NAMESPACE}-${JOB_NAME_HASH}"
 
 # create vnet
 vnet_arm_template_file="/tmp/01_vnet.json"
-run_command "curl -L https://raw.githubusercontent.com/openshift/installer/master/upi/azure/01_vnet.json -o ${vnet_arm_template_file}"
+vnet_template_url="https://raw.githubusercontent.com/openshift/installer/master/upi/azure/01_vnet.json"
```

Comment:
I guess we can get the "01_vnet.json" at "/var/lib/openshift-install/upi/${CLUSTER_TYPE}/" from upi-installer:4.12 executor pod itself, not have to download it.